pybamm/models/full_battery_models/lithium_ion/base_lithium_ion_model.py

```diff
@@ -159,13 +159,17 @@ def set_degradation_variables(self):
 
         # Lithium lost to side reactions
         # Different way of measuring LLI but should give same value
-        n_Li_lost_sei = self.variables["Loss of lithium to SEI [mol]"]
-        n_Li_lost_reactions = n_Li_lost_sei
-        if "negative electrode" in domains:
+        n_Li_lost_neg_sei = self.variables["Loss of lithium to negative SEI [mol]"]
+        n_Li_lost_pos_sei = self.variables["Loss of lithium to positive SEI [mol]"]
+        n_Li_lost_reactions = n_Li_lost_neg_sei + n_Li_lost_pos_sei
```

Comment:
Could you include these in the loop below, or is there a reason to keep them separate?

Reply:
It has to be separate because SEI can also grow on lithium metal electrodes, but that's not included in the loop because `domains` is defined on line 109 to be only whole cell domains.